### DISCUSSION TITLE
Resources should contain a different prefix than nova-api

### DISCUSSION THREAD

**Discussion by stephan-v** [laravel/nova-issues#5489] [Q&A] [answered]
Try adding a `style` resource a `dashboard` resource or a `metric` resource. All of these will fail because Laravel Nova already contains routes for these.

For example a route for the styles which is registered in a clean install of Nova without resources looks as follows:

`nova-api/styles/{style}`

Obviously this will clean when registering a `style` resource because the general resource route of `nova-api/{resource}` will be transformed into `nova-api/styles` as well leading to 404's.

Now I now the workaround would be to rename my resource to something else and specify the Style model manually but just using a different prefix for the routes would ensure that you would never run into this issue at all.

**Comment by crynobone** [comment, marked as answer]
This has been defined in our documentation under Reserved Resource Name: https://nova.laravel.com/docs/4.0/resources/#defining-resources

**Comment by stephan-v** [comment]
So why wouldn't you just create a prefix for the resource routes to ensure you don't get clashes?
